docs/docsite/rst/reference_appendices/YAMLSyntax.rst

```diff
@@ -107,6 +107,9 @@ While in the above ``>`` example all newlines are folded into spaces, there are
         d
           e
         f
+
+Alternatively, it can be enforced by including newline ``\n`` characters::
+
     same_as: "a b\nc d\n  e\nf\n"
```

Comment:
```suggestion
    fold_some_newlines: "a b\nc d\n  e\nf\n"
```

Reply:
@sankur-codes
Apologies for the extra change request, but keeping the key name the same makes sense when the code blocks are split.

Reply:
Created a commit to maintain the equivalence and preserve the context of "same". Thanks!

Reply:
Great, thank you.